client/my-sites/domains/domain-management/dataviews/components/bulk-update-notice.tsx

```diff
@@ -63,39 +71,59 @@ export const BulkUpdateNotice = () => {
 			} );
 		}
 		handleRestartDomainStatusPolling();
-	};
+	}, [ deleteBulkActionStatus, handleRestartDomainStatusPolling ] );
 
-	// completed jobs can be announced
-	return completedJobs
-		.filter( ( job ) => ! dismissedJobs.includes( job.id ) )
-		.map( ( job ) => {
-			if ( job.failed.length ) {
-				return (
-					<NoticeWithDetails
-						key={ job.id }
-						status="is-error"
-						message={ translate( 'Some domain updates were not successful.' ) }
-						domains={ job.failed }
-						jobId={ job.id }
-						handleDismissNotice={ handleDismissNotice }
-					/>
-				);
-			}
+	const [ shownSuccessNotices, setShownSuccessNotices ] = useState< string[] >( [] );
+	const unshownJobs = useMemo(
+		() => completedJobs.filter( ( job ) => ! shownSuccessNotices.includes( job.id ) ),
+		[ completedJobs, shownSuccessNotices ]
+	);
 
-			const message =
-				job.success.length > 1
-					? translate( 'Bulk domain updates finished successfully.' )
-					: translate( 'Domain update finished successfully.' );
+	useEffect( () => {
+		// completed jobs can be announced
+		unshownJobs
+			.map( ( job ) => {
+				setShownSuccessNotices( shownSuccessNotices.concat( [ job.id ] ) );
+				return job;
+			} )
+			.map( ( job ) => {
+				if ( job.failed.length ) {
+					errorNotice(
+						<div style={ { display: 'flex', alignItems: 'center', gap: '8px' } }>
+							{ translate( 'Some domain updates were not successful.' ) }
+							<StatusPopover
+								position="bottom"
+								popoverTargetElement={
+									<div style={ { color: 'var(--color-text-inverted)', fontSize: '0.875rem' } }>
+										{ translate( 'Domains list' ) }{ ' ' }
+									</div>
+								}
+							>
+								<div className="domains-table-bulk-actions-notice-popover">
+									{ job.failed.map( ( domain ) => (
+										<p key={ domain }> { domain } </p>
+									) ) }
+								</div>
+							</StatusPopover>
+						</div>
+					);
+
+					return;
+				}
 
-			return (
-				<NoticeWithDetails
-					key={ job.id }
-					status="is-success"
-					message={ message }
-					domains={ job.success }
-					jobId={ job.id }
-					handleDismissNotice={ handleDismissNotice }
-				/>
-			);
-		} );
+				const message =
+					job.success.length > 1
+						? translate( 'Bulk domain updates finished successfully.' )
+						: translate( 'Domain update finished successfully.' );
+
+				successNotice( message );
+			} );
+		deleteBulkActionStatusWhenShown();
+	}, [ unshownJobs ] );
```

Comment:
Doesn't include all the dependencies. 
But if I include the complete list, it triggers the effect infinitely.

Reply:
Yeah, sometimes our business logic requires only one dependency. In that case, it's OK to ignore the linting error. I have had similar linting complaints before, but I ignored them with the comment explaining why.

Reply:
I left the above comment without looking into the codebase changes. Let me check how the code looks like.